main.py

```diff
@@ -43,7 +43,7 @@ def main():
 
     experiment_path = config["experiment_path"]
     n_repeats = config["n_repeats"]
-
+    print(scenario_params_list)
```

Comment:
Why not log this? Or just remove?

Reply:
It was for debugging purpose, I forgot to remove it